.github/workflows/ci.yml

```diff
@@ -0,0 +1,288 @@
+name: Rust
+on:
+  push:
+    branches:
+      - master
+      - ci # TODO: remove later
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Please add a `v` in front of that regex. All tags are currently in the scheme `v0.5.0` not `0.5.0`.

Reply:
Done.